index.js

```diff
@@ -1,100 +1,71 @@
-const defaultOptions = require('./defaultOptions.js');
+const defaultOptions = require('./lang/defaultOptions.js');
```

Comment:
No need for default custom messages since users already have the possibility to customize them (same logic as for the word)

Reply:
Lang option deleted